test.py

```diff
@@ -89,19 +61,23 @@ def parse_args():
     },
     'board_config':
     {
-        'swap_left_and_right_cameras': True, # True for 1097 (RPi Compute) and 1098OBC (USB w/onboard cameras)
-        'left_fov_deg': 69.0, # Same on 1097 and 1098OBC
-        'left_to_right_distance_cm': 9.0, # Distance between stereo cameras
-        'left_to_rgb_distance_cm': 2.0 # Currently unused
+        'swap_left_and_right_cameras': False,
```

Comment:
Any reason comments were dropped and the default values were changed? We want the defaults to line up with the BW1097, which is the most popular edition of DepthAI.